.github/workflows/build-and-test.yml

```diff
@@ -2897,12 +2909,23 @@ jobs:
     runs-on: ubuntu-20.04
     needs: [build-lib-LinuxArmv7, swig_Linux]
     strategy:
+      fail-fast: false
       matrix:
-        python-version: [3.7]
+        include:
+          - python-version: "3.7"
+            system-raspbian: multistrap-raspbian-buster
+            multistrap-config: multistrap_raspbian_buster.conf
+          # Only experimental support for python 3.8
```

Comment:
remove the command it will confuse people. We’ll take care of py38 in another pr.

Reply:
done